.php-cs-fixer.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
We'll also need to update the `.gitattributes` accordingly so the changed file is not exported, and possibly also the pre-release Composer script.